src/main/resources/io/jenkins/plugins/pipelinegraphview/PipelineGraphViewAction/index.jelly

```diff
@@ -2,14 +2,26 @@
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:p="/lib/pipeline-graph-view">
-  <l:layout title="${%Graph} [${it.buildDisplayName}]" type="one-column">
+  <l:layout title="${%Graph} [${it.fullBuildDisplayName}]" type="one-column">
     <l:main-panel>
       <div class="jenkins-app-bar">
         <div class="jenkins-app-bar__content">
           <h1>
-            <l:icon alt="${it.iconColor.description}" class="${it.buildStatusIconClassName} icon-xlg"
-                    tooltip="${it.iconColor.description}"/>
+            <l:icon alt="${it.iconColor.description}"
+              class="${it.buildStatusIconClassName} icon-xlg"
+              tooltip="${it.iconColor.description}" />
+            ${it.fullProjectName}
+            <j:if test="${it.previousBuildNumber!=null}">
+              <a href="../../${it.previousBuildNumber}/pipeline-graph" style="text-decoration:none; color:inherit;">
+                <l:icon class="symbol-chevron-back-outline plugin-ionicons-api icon-md" tooltip="${%Previous Build}" />
+              </a>
+            </j:if>
             ${it.buildDisplayName}
+            <j:if test="${it.nextBuildNumber!=null}">
+              <a href="../../${it.nextBuildNumber}/pipeline-graph" style="text-decoration:none; color:inherit;">
```

Comment:
could we use CSS from a stylesheet instead of inline please? If we ever want to change the styling it would be in one place instead of 4
https://github.com/jenkinsci/pipeline-graph-view-plugin/blob/main/src/main/frontend/pipeline-graph-view/app.scss

Reply:
I thought they didn't look very good left blue, but happy to leave it.
![image](https://user-images.githubusercontent.com/551572/218987523-2254111b-d491-48fa-87da-e5db5a801fde.png)

Reply:
@timja any opinion?

Reply:
I think the inherit color is better, no need to draw attention to these, but we can easily change in the future

Reply:
I haven't had any success using CSS . I've pushed what I have locally so maybe someone else can see what I'm missing.

Reply:
works after what I pushed